Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/person/core/PersonCertificatesStorageTest.kt

```diff
@@ -113,6 +114,97 @@ class PersonCertificatesStorageTest : BaseTest() {
+        createInstance().apply {
+            setBoosterNotifiedAt(personIdentifier1, Instant.EPOCH)
+            setDccReissuanceNotifiedAt(personIdentifier1, Instant.EPOCH)
+            setGStatusNotifiedAt(personIdentifier1, Instant.EPOCH)
+            fakeDataStore[PERSONS_SETTINGS_MAP].toString().toComparableJsonPretty() shouldBe """
+                {
+                	"settings": {
+                		"{\"dateOfBirth\":\"01.10.2020\",\"familyNameStandardized\":\"lN\",\"givenNameStandardized\":\"fN\"}": {
+                			"lastSeenBoosterRuleIdentifier": null,
+                			"lastBoosterNotifiedAt": 0,
+                			"showDccReissuanceBadge": true,
+                			"lastDccReissuanceNotifiedAt": 0,
+                            "showAdmissionStateChangedBadge": true,
+                            "lastAdmissionStateNotifiedAt": 0
+                		}
+                	}
+                }
+            """.trimIndent()
+                .toComparableJsonPretty()
+        }
+    }
+
+    @Test
+    fun `dismiss G status for a person with no settings`() = runBlockingTest {
+        createInstance().apply {
+            dismissGStatusBadge(personIdentifier1)
+            fakeDataStore[PERSONS_SETTINGS_MAP].toString().toComparableJsonPretty() shouldBe """
+                {
+                	"settings": {
+                		"{\"dateOfBirth\":\"01.10.2020\",\"familyNameStandardized\":\"lN\",\"givenNameStandardized\":\"fN\"}": {
+                			"lastSeenBoosterRuleIdentifier": null,
+                			"lastBoosterNotifiedAt": null,
+                			"showDccReissuanceBadge": false,
+                			"lastDccReissuanceNotifiedAt": null,
+                            "showAdmissionStateChangedBadge": false,
+                            "lastAdmissionStateNotifiedAt": null
+                		}
+                	}
+                }
+            """.trimIndent()
+                .toComparableJsonPretty()
+        }
+    }
+
+    @Test
+    fun `dismiss G status for a person that has settings`() = runBlockingTest {
+        createInstance().apply {
+            setGStatusNotifiedAt(personIdentifier1, Instant.EPOCH)
+            dismissGStatusBadge(personIdentifier1)
+            fakeDataStore[PERSONS_SETTINGS_MAP].toString().toComparableJsonPretty() shouldBe """
+                {
+                	"settings": {
+                		"{\"dateOfBirth\":\"01.10.2020\",\"familyNameStandardized\":\"lN\",\"givenNameStandardized\":\"fN\"}": {
+                			"lastSeenBoosterRuleIdentifier": null,
+                			"lastBoosterNotifiedAt": null,
+                			"showDccReissuanceBadge": false,
+                			"lastDccReissuanceNotifiedAt": null,
+                            "showAdmissionStateChangedBadge": false,
+                            "lastAdmissionStateNotifiedAt": 0
+                		}
+                	}
+                }
+            """.trimIndent()
+                .toComparableJsonPretty()
+        }
+    }
+
```

Comment:
Please adjust  the formatting :D? I see the two new fields are misaligned

Reply:
Hmm, that's weird. It looks alright locally. I will try to add a tab indentation and see what happens.